src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -146,6 +147,10 @@ private function renderProductForm(FormInterface $productForm, ?int $productId =
             'helpLink' => $this->generateSidebarLink('AdminProducts'),
             'isMultiShopContext' => $isMultiShopContext,
             'editable' => $this->isGranted(PageVoter::UPDATE, self::PRODUCT_CONTROLLER_PERMISSION),
+            //@todo: hardcoded. Make configurable?
+            'combinationLimitChoices' => [10, 20, 50, 100],
+            'combinationsLimit' => 10,
```

Comment:
We made this configurable for Order page products display, so it makes sense to have some configuration here as well (but later)

Reply:
Could you put in constants please ?